<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scmSkyCultureDialog</class>
 <widget class="QWidget" name="scmSkyCultureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000000</width>
    <height>999999</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBar" name="titleBar">
     <property name="title" stdset="0">
      <string>Sky Culture Maker</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="Content">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabs">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>800</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <widget class="QWidget" name="overviewTab">
         <attribute name="title">
          <string>Overview</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="currentSCNameLbl">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Name of the Sky Culture</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="skyCultureNameTE">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>32</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="constellationsLbl">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Constellations:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="constellationsList">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>64</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
              QListWidget, QListWidget::viewport {
                  margin: 0px;
                  padding: 0px;
                  border: none;
                  outline: none;
              }
              QListWidget::item {
                  margin: 0px;
                  padding: 4px;
                  border: none;
                  outline: none;
              }
             </string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="overviewConstellationBtnLayout">
            <item>
             <widget class="QPushButton" name="AddConstellationBtn">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Add Constellation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="RemoveConstellationBtn">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Remove Constellation</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="overviewSCBtnLayout">
            <item>
            <widget class="QLabel" name="selectLicenseLbl">
              <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
              </sizepolicy>
              </property>
              <property name="font">
              <font>
                <pointsize>14</pointsize>
              </font>
              </property>
              <property name="text">
              <string>License of the Sky Culture</string>
              </property>
              <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
            </widget>
            </item>
            <item>
            <widget class="QComboBox" name="licenseCB">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
              </sizepolicy>
              </property>
            </widget>
            </item>
            <item>
             <widget class="QPushButton" name="SaveSkyCultureBtn">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Save Sky Culture</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="descriptionTab">
          <attribute name="title">
            <string>Description</string>
          </attribute>
          <layout class="QVBoxLayout" name="descriptionTabLayout">
            <item>
            <widget class="QScrollArea" name="descriptionScrollArea">
              <property name="widgetResizable">
              <bool>true</bool>
              </property>
              <widget class="QWidget" name="descriptionScrollContent">
              <layout class="QVBoxLayout" name="descriptionLayout">
                <property name="spacing">
                <number>10</number>
                </property>
                <property name="leftMargin">
                <number>10</number>
                </property>
                <property name="topMargin">
                <number>10</number>
                </property>
                <property name="rightMargin">
                <number>10</number>
                </property>
                <property name="bottomMargin">
                <number>10</number>
                </property>

                <!-- Authors -->
                <item>
                <widget class="QLabel" name="authorsLbl">
                  <property name="text">
                  <string>Authors:</string>
                  </property>
                </widget>
                </item>
                <item>
                <widget class="QTextEdit" name="authorsTE"/>
                </item>

                <!-- Culture Description -->
                <item>
                <widget class="QLabel" name="cultureDescriptionLbl">
                  <property name="text">
                  <string>Culture Description:</string>
                  </property>
                </widget>
                </item>
                <item>
                <widget class="QTextEdit" name="cultureDescriptionTE"/>
                </item>

                <!-- About -->
                <item>
                <widget class="QLabel" name="aboutLbl">
                  <property name="text">
                  <string>About:</string>
                  </property>
                </widget>
                </item>
                <item>
                <widget class="QTextEdit" name="aboutTE"/>
                </item>

                <!-- Geo-Region -->
                <item>
                <widget class="QLabel" name="geoRegionLbl">
                  <property name="text">
                  <string>Geo-Region:</string>
                  </property>
                </widget>
                </item>
                <item>
                <widget class="QTextEdit" name="geoRegionTE"/>
                </item>

                <!-- Sky -->
                <!-- Sky -->
                <item>
                <layout class="QHBoxLayout">
                  <item>
                    <widget class="QLabel" name="skyLbl">
                      <property name="text">
                        <string>Sky</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QPushButton" name="pushButton">
                      <property name="enabled">
                        <bool>false</bool>
                      </property>
                      <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                        </sizepolicy>
                      </property>
                      <property name="cursor">
                        <cursorShape>WhatsThisCursor</cursorShape>
                      </property>
                      <property name="contextMenuPolicy">
                        <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
                      </property>
                      <property name="toolTip">
                        <string>what can the User see here</string>
                      </property>
                      <property name="autoFillBackground">
                        <bool>false</bool>
                      </property>
                      <property name="icon">
                        <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
                      </property>
                      <property name="autoDefault">
                        <bool>false</bool>
                      </property>
                      <property name="default">
                        <bool>false</bool>
                      </property>
                      <property name="flat">
                        <bool>true</bool>
                      </property>
                    </widget>
                  </item>
                </layout>
              </item>
              <item>
                <widget class="QTextEdit" name="skyTE"/>
              </item>


                <!-- Moon & Sun -->
                <item>
                <widget class="QLabel" name="moonSunLbl">
                  <property name="text">
                  <string>Moon &amp; Sun:</string>
                  </property>
                </widget>
                </item>
                <item>
                <widget class="QTextEdit" name="moonSunTE"/>
                </item>

                <!-- Planets -->
                <item>
                <widget class="QLabel" name="planetsLbl">
                  <property name="text">
                  <string>Planets:</string>
                  </property>
                </widget>
                </item>
                <item>
                <widget class="QTextEdit" name="planetsTE"/>
                </item>

                <!-- Zodiac / Lunar System -->
                <item>
                <widget class="QLabel" name="zodiacLbl">
                  <property name="text">
                  <string>Zodiac/ Lunar System:</string>
                  </property>
                </widget>
                </item>
                <item>
                <widget class="QTextEdit" name="zodiacTE"/>
                </item>

                <!-- Milky Way -->
                <item>
                <widget class="QLabel" name="milkyWayLbl">
                  <property name="text">
                  <string>Milky Way:</string>
                  </property>
                </widget>
                </item>
                <item>
                <widget class="QTextEdit" name="milkyWayTE"/>
                </item>

                <!-- Other Celestial Objects -->
                <item>
                <widget class="QLabel" name="otherObjectsLbl">
                  <property name="text">
                  <string>Other Celestial Objects:</string>
                  </property>
                </widget>
                </item>
                <item>
                <widget class="QTextEdit" name="otherObjectsTE"/>
                </item>

                 <!-- Constellations -->
                <item>
                <widget class="QLabel" name="constellationsDescLbl">
                  <property name="text">
                  <string>Constellations:</string>
                  </property>
                </widget>
                </item>
                <item>
                <widget class="QTextEdit" name="constellationsDescTE"/>
                </item>

                <!-- References -->
                <item>
                <widget class="QLabel" name="referencesLbl">
                  <property name="text">
                  <string>References:</string>
                  </property>
                </widget>
                </item>
                <item>
                <widget class="QTextEdit" name="referencesTE"/>
                </item>

                <!-- Acknowledgements -->
                <item>
                <widget class="QLabel" name="acknowledgementsLbl">
                  <property name="text">
                  <string>Acknowledgements:</string>
                  </property>
                </widget>
                </item>
                <item>
                <widget class="QTextEdit" name="acknowledgementsTE"/>
                </item>

                <!-- Classification -->
                <item>
                  <widget class="QLabel" name="classificationLbl">
                    <property name="text">
                    <string>Classification of the Sky Culture</string>
                    </property>
                  </widget>
                  </item>
                  <item>
                  <widget class="QComboBox" name="classificationCB">
                    <property name="font">
                    <font>
                      <pointsize>13</pointsize>
                    </font>
                    </property>
                  </widget>
                </item>

              </layout>
              </widget>
            </widget>
          </item>
        </layout>
        </widget>
        <widget class="QWidget" name="boundariesTab">
         <attribute name="title">
          <string>Boundaries</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="commonNamesTab">
         <attribute name="title">
          <string>Common Names</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="geoLocationTab">
         <attribute name="title">
          <string>Geolocation</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
          <spacer name="horizontalSpacerLeft">
            <property name="orientation">
            <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
            <size>
              <width>40</width>
              <height>20</height>
            </size>
            </property>
          </spacer>
          </item>
          <item>
          <widget class="QLabel" name="infoLbl">
            <property name="font">
            <font>
              <weight>75</weight>
              <bold>true</bold>
            </font>
            </property>
            <property name="text">
            <string/>
            </property>
            <property name="alignment">
            <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
            <bool>true</bool>
            </property>
          </widget>
          </item>
          <item>
          <spacer name="horizontalSpacerRight">
            <property name="orientation">
            <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
            <size>
              <width>40</width>
              <height>20</height>
            </size>
            </property>
          </spacer>
          </item>
        </layout>
        </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QFrame</extends>
   <header location="global">Dialog.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/SkyCultureMaker.qrc"/>
 </resources>
 <connections/>
</ui>
